src/views/endorse-company-closure-certificate.njk

```diff
@@ -41,12 +41,15 @@
 
       <h2 id="declaration-heading" class="govuk-heading-m">Declaration of {{ approvalModel.officerType }}s</h2>
 
-      <p id="declaration" class="govuk-body">I/We as {{ approvalModel.officerType }}s / the majority of {{ approvalModel.officerType }}s apply for this {% if approvalModel.officerType === 'member' %} LLP {% else %} company {% endif %} to be struck off the Register
+      <p id="declaration-paragraph" class="govuk-body">I/We as {{ approvalModel.officerType }}s / the majority of {{ approvalModel.officerType }}s apply for 
+        this {% if approvalModel.officerType === 'member' %} LLP {% else %} company {% endif %} to be struck off the Register
         and declare that:
```

Comment:
Please move this to the previous line

Reply:
Do you mean no line break between heading and paragraph?? Move it to 43? Will I do this to the rest?